source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Domain/Dtos/ChargeCommands/Validation/InputValidation/ValidationRules/NumberOfPointsMatchTimeIntervalAndResolutionRuleTests.cs

```diff
@@ -39,6 +39,8 @@ public class NumberOfPointsMatchTimeIntervalAndResolutionRuleTests
         [InlineAutoMoqData(Resolution.PT1H, 2022, 10, 29, 22, 2022, 10, 30, 23, 25, "switching to Normal Time must be supported")]
         [InlineAutoMoqData(Resolution.P1D, 2022, 10, 29, 22, 2022, 10, 30, 23, 1, "switching to Normal Time must be supported")]
         [InlineAutoMoqData(Resolution.P1M, 2022, 10, 29, 22, 2022, 11, 30, 23, 1, "switching to Normal Time must be supported")]
+        [InlineAutoMoqData(Resolution.P1M, 2022, 3, 26, 23, 2022, 3, 27, 22, 1, "switching to Daylight Saving Time, with irregular price series must be supported")]
+        [InlineAutoMoqData(Resolution.P1M, 2022, 10, 29, 22, 2022, 10, 30, 23, 1, "switching to Normal Time, with irregular price series must be supported")]
```

Comment:
Minor thing, but I would like endTime to reflect how an irregular price series should look like.

```suggestion
        [InlineAutoMoqData(Resolution.P1M, 2022, 3, 26, 23, 2022, 3, 31, 22, 1, "switching to Daylight Saving Time, with irregular price series must be supported")]
        [InlineAutoMoqData(Resolution.P1M, 2022, 10, 29, 22, 2022, 10, 31, 23, 1, "switching to Normal Time, with irregular price series must be supported")]
```